src/views/ManageList.tsx

```diff
@@ -76,6 +85,43 @@ export function ManageList({ listPath }: Props) {
 		}
 	};
 
+	const handleInvite = async (
+		e: FormEvent<HTMLFormElement>,
+		listPath: string,
+	) => {
+		console.log("Button clicked! Inviting user!");
+		e.preventDefault();
+
+		const trimmedEmailName = validateTrimmedString(emailName);
+
+		if (!trimmedEmailName) {
+			toast.error(
+				"Oops! Email cannot be empty or just spaces. Please enter a valid email.",
+			);
+			return;
+		}
+
+		if (currentUser === null) {
+			toast.error("You are not logged in! Cannot invite.");
+			return;
+		}
+
+		try {
+			await toast.promise(shareList(listPath, currentUser, trimmedEmailName), {
+				loading: "sharing list with existing user",
```

Comment:
> I'm so glad we got some more practice with React Hot Toast this week!

What do you think about changing the messages? Something like `Successfully invited ${emailName} to your list!` for success? Something like `Oops! Failed to invite ${emailName} to your list. Please try again!` for error?

The important parts we got done are:
A. The toasts are working!
B. The variable shows up in string interpolation. 

I really like changing the messages to be a bit more descriptive! It would allow another way for the user to see like "who did i just invite?".

Reply:
Nothing wrong with answering questions! We can all work together. 😄

Reply:
Yes! I like that idea too. I was playing around in the useAuth.tsx file, and I noticed we have access to user.name and can invoke the list owner/user's name into our notifications, but it would be so cool if we could insert the invited user's name instead of their email !!!!